### DISCUSSION TITLE
Query about EXT_lights_image_based extension

### DISCUSSION THREAD

**Discussion by Jarreddebeer** [donmccurdy/glTF-Transform#496] [General]
Hi there,

Really enjoying your library, kudos and thank you for it.

Is there an extension floating around for [EXT_lights_image_based](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Vendor/EXT_lights_image_based/README.md)?

I noticed the other 3 EXT extensions are implemented, and am curious if this 4th one is available. If not perhaps I can try my hand at an implementation, but I don't have an intuition for how much work is involved, would appreciate your advice.

Thanks

**Comment by donmccurdy** [comment]
I don't have an implementation of EXT_lights_image_based currently. I've been delaying on that until seeing where adoption is headed. Notably, there's an alternative proposal in https://github.com/KhronosGroup/glTF/pull/1956. If you wanted to try implementing one or the other, I think these two existing extensions would be good references:

- https://github.com/donmccurdy/glTF-Transform/tree/ca187965eec975fc7027d473be39d84b0cff5b29/packages/extensions/src/khr-materials-transmission
- https://github.com/donmccurdy/glTF-Transform/tree/ca187965eec975fc7027d473be39d84b0cff5b29/packages/extensions/src/khr-lights-punctual

**Reply by Jarreddebeer** [reply]
Thank you, I wasn't aware of the alternate proposal.. i'm pretty new to the space. Perhaps i'll also wait and see, but will learn from your links in the mean time, appreciate it
